h2/utilities.py

```diff
@@ -40,6 +42,11 @@
     b'proxy-authorization', u'proxy-authorization',
 ])
 
+if sys.version_info[0] == 2:  # pragma: no cover
```

Comment:
If you change this comment to `# Python 2.X`, it will still exclude the coverage, and be a bit clearer.